### PR TITLE
Add Node School's Git, SQLZoo and Free Code Camp

### PR DESCRIPTION
**Pull request by QuincyLarson**

### CODE REVIEW

**Comment by onebree** [comment]
@vhf Rather than having FCC posted twice on the list, I think we should have a "Full Stack web development" section. While Full Stack is a bit of a buzz word, I think it can include the following:

- Free Code Camp (new, JS focused)
- Odin Project (existing, Ruby/Rails focused)

**Comment by vhf** [comment]
I agree on the idea @onebree but I'm not sure about the section name you suggest. I'll think about it.

**Comment by QuincyLarson** [comment]
Hi @vhf did you have time to think about this? Please let me know if you have any questions about Free Code Camp or the other resources I recommended adding.

File: free-programming-interactive-tutorials-en.md [hunk 1]
```diff
@@ -77,6 +77,7 @@
 ### Git
 
 * [Githug](https://github.com/Gazler/githug) (Tutorial in shell)
+* [Git-it](https://github.com/jlord/git-it) (Tutorial in shell)
```
Comment (onebree) [inline free-programming-interactive-tutorials-en.md]: Should be first item in list (above githug)

**Pull request closed without merging**